packages/app/cypress/e2e/runner/reporter-ct-mount-hover.cy.ts

```diff
@@ -0,0 +1,51 @@
+import type { fixtureDirs } from '@tooling/system-tests'
+
+type ProjectDirs = typeof fixtureDirs
+
+const PROJECTS: {projectName: ProjectDirs[number], test: string}[] = [
+  { projectName: 'angular-15', test: 'app.component' },
```

Comment:
Let's change this to `angular-14`, Angular 15 is in beta so it shouldn't be the version we test our stable versions against

Reply:
This has been updated in the latest commit